Backend/Workez.API/Controllers/ShiftController.cs

```diff
@@ -0,0 +1,60 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Workez.Application.DTOs;
+using Workez.Application.Services;
+using Workez.Infrastructure.Persistence;
+
+namespace Workez.API.Controllers;
+
+[ApiController]
```

Comment:
Do you have developed the authorization middleware? If that's the case you should add

```suggestion
[ApiController]
[Authorize]
```

This requires authorization to all of the controller endpoints. You can also add that attribute per endpoint instead of all of them.

If you don't have yet the authorization, do not add until it is developed and working.